<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>АИС Футбольных клубов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="menuLayout">
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="text">
         <string>Выйти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clubButton">
        <property name="text">
         <string>Клубы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playerButton">
        <property name="text">
         <string>Игроки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="coachButton">
        <property name="text">
         <string>Тренеры</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tournamentButton">
        <property name="text">
         <string>Турниры</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
